WMIAdventure/frontend/src/js/api/endpoints/UserProfilesEndpoints.js

```diff
@@ -9,7 +9,8 @@ class UserProfilesEndpoints {
     static userById = (userId) => `${UserProfilesEndpoints.main}${userId}/`;
     // If we have more decks that one the deck number will also be needed
     static userIdDeck = (userId) => `${UserProfilesEndpoints.main}${userId}/decks/1/`;
-    static userIdLevel = (userId) => `${UserProfilesEndpoints.main}${userId}/level/`
+    static userIdLevel = (userId) => `${UserProfilesEndpoints.main}${userId}/level/`;
+    static userCards = (userId) => `${UserProfilesEndpoints.main}${userId}/cards/`;
```

Comment:
ok, bo tamten widok to był dla dwóch decków, teraz jest `${UserProfilesEndpoints.main}${userId}/cards/` jako jeden deck danego usera

Reply:
to w sumie nawet nie jest deck tylko wszystkie jego karty